backend/pkg/transport/packet/info/packet.go

```diff
@@ -29,3 +32,14 @@ func NewPacket(id abstraction.PacketId) *Packet {
 func (packet *Packet) Id() abstraction.PacketId {
 	return packet.id
 }
+
+// GetBoardId returns the board id
+func (p *Packet) GetBoardId() string {
+	return fmt.Sprint(p.BoardId)
+}
+
+// GetTimestamp returns the timestamp as a time.Time object
+func (p *Packet) GetTimestamp() time.Time {
+	t := p.Timestamp
+	return time.Date(int(t.Year), time.Month(int(t.Month)), int(t.Day), int(t.Hour), int(t.Minute), int(t.Second), int(t.Counter), time.UTC)
+}
```

Comment:
make this return `packet.Timestamp` and then have a method for `packet.Timestamp` with the signature `func (timestamp packet.Timestamp) ToTime() time.Time` which does this